app/helpers/view_helper.rb

```diff
@@ -131,6 +131,10 @@ def provider_user_name(provider_user)
     end
   end
 
+  def percent_of(number)
```

Comment:
Do you need to pass two args into this method now it's a view helper? e.g.

```
def percent_of(numerator, denominator)
    numerator.to_f / denominator * 100.0
end
```

Reply:
Apologies, it worked implicitly by calling it in our private method `count.percent_of(total)`which calls `.to_f` on the count and divides it by `total *100`. Therefore, I didn't think to change it.

I think that your way of writing it though is more explicit and is probably more useful for future applications. Have updated now.